src/1-foundations/2-homotopy-type-theory/01-types-are-higher-groupoids.rzk.md

```diff
@@ -5,3 +5,141 @@ This is a literate Rzk file:
 ```rzk
 #lang rzk-1
 ```
+
+Groupoids are categories in which all morphisms are isomorphisms.
+Alternativerly, groupoids can be viewed as a generalization of groups, where not all pairs of elements can be composed
+(but the group laws for the operation hold).
+
+## Path symmetry
+!!! lemma "Lemma 2.1.1."
+    For every type $A$ and every $x, y : A$ there is a function $(x = y) \to (y = x)$`
+    denoted $p \mapsto p^{-1}$, such that $refl_x^{-1} \equiv refl_x$ for each $x : A$. We call $p^{-1}$ the inverse of $p$.
+
+```rzk
+#def path-sym
+  (A : U)
+  (x y : A)
+  : (x = y) → (y = x)
+  := path-ind 
+    A 
+    (\ x' y' _ → y' = x') 
+    (\ z → refl) 
+    x y
+```
+
+## Path concatenation
+!!! lemma "Lemma 2.1.2."
+    For every type $A$ and every $x, y, z : A$ there is a function $(x = y) \to (y = z) \to (x = z)$,
+    written $p \mapsto q \mapsto p \cdot q$`, such that $refl_x \cdot refl_x \equiv refl_x$` for any $x : A$. 
```

Comment:
These backticks (`) make it render incorrectly

Reply:
fixed, thank you!